File: Platazi_Learn/POO/Python/UberX.py
from Car import Car

class uberX(Car):
    brand = str
    model = str

    def __init__(self, license, driver, brand, model):
        super().__init__(license, driver)
        self.brand = brand
        self.model = model
    
    def printDataUberX(self):
        super().printDataCar()
        print("Model: ", self.model)
        print("Brand: ", self.brand, "\n")

File: Platazi_Learn/POO/PHP/UberX.php
<?php

require_once ('car.php');

class UberX extends Car{
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
    function PrintDataUberX() {
        parent::PrintDataCar();
        echo "    Marca: $this->brand 
        Modelo: $this->model
        ";
    }
}